middleware.ts

```diff
@@ -0,0 +1,13 @@
+import { initAuth0, withMiddlewareAuthRequired } from '@auth0/nextjs-auth0/edge';
+
+const isDevEnvironment = process.env.NODE_ENV === 'development';
```

Comment:
Based on https://nextjs.org/docs/messages/non-standard-node-env, we might want this for 'test' too.  Any idea why our tests are working without this?